### DISCUSSION TITLE
Table hyperlinks or GUI tooltip with "Open in new tab", which is better?

### DISCUSSION THREAD

**Discussion by Autr** [directus/directus#17685] [Feature Requests]
### Summary

Hello,

I noticedwith a few of projects recently some friction onboarding people to begin using Directus. I'd think: how could this possibly be when their previous system was a pile of laggy Wordpress sludge? 

... Well in each case its with the person who will be editing _a lot of entries_. And it takes me back to a frustration I first had when using Directus - and the first comment someone gave me back in 2019 (when using V8):

"Oh great, I can't event right-click open new tab"

And come to think of it, this is crucial... 

### Basic Example

<img width="631" alt="Screenshot 2023-03-02 at 23 17 20" src="https://user-images.githubusercontent.com/56603487/222571759-c579a3b2-17ec-45fe-b7cb-20f3a1c9a1b1.png">

### Motivation

I think something like this gets missed because we might develop a workaround and move on (ie. copy-paste URL into new tab or CMD/Ctrl + Click), or because developers rarely get stuck into manual data-entry. Best UX feedback is just watching someone use a tool for the first time!

### Detailed Design

My guess is that A tags aren't used to encapsulate rows or cells because there are components filled with block level elements like DIVs. I can see two ways to do this:

1) replace all internal DIVs within components used inside cells with SPANs and add necessary block, flex etc classes, then encapsulate those cells in A tags. However this might be tricky if some components used within cells come from an external library.
2) reuse the tooltip component that's used elsewhere, and perhaps also add some more options whilst doing it, ie. _Open in New Tab, Edit Item, Select / Deselect, Select all / Deselect All, Archive Item, Delete Item_

### Requirements List

Some indication from maintainers of preference for which route to take; encapsulate in A tag or generate tooltip - then I'll do it.

### Drawbacks

1) A tags -> the need to replace all block level elements within components used within cells
2) Tooltip -> would need some thought if there is an existing pattern or methodology for tooltips

### Alternatives

No alternatives - just the low-level annoyance of data-entryists :P 

### Adoption Strategy

N/A

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
